test/tests/functional/pbs_alps_request_tunables.py

```diff
@@ -0,0 +1,150 @@
+    and alps_release_jitter
+    """
+
+    def setUp(self):
+        machine = self.du.get_platform()
+        if not machine.startswith('cray') or machine == "craysim":
+            self.skipTest("Test suite only meant to run on a Cray")
+        TestFunctional.setUp(self)
+
+    def get_epoch(self, msg):
+        a = time.strptime(msg[:19], "%m/%d/%Y %H:%M:%S")
+        return int(time.mktime(a))
+
+    def test_alps_release_interval(self):
+        """
+        Set alps_release_interval to a higher value and then notice that
+        subsequest reservation cancellation requests are made atleast
+        after the set interval.
+        """
+        ari = 4.298
+        self.mom.add_config({'$alps_release_interval': ari})
+
+        # submit a job and then delete it after it starts running
+        start_time = time.time()
+        j1 = Job(TEST_USER)
+        jid1 = self.server.submit(j1)
+        self.server.expect(JOB, {ATTR_state: 'R'}, id=jid1)
+        self.server.delete(jid1)
+
+        # Look for a message that confirms that reservation is deleted
+        self.mom.log_match("%s;ALPS reservation cancelled" % jid1,
+                           starttime=start_time, max_attempts=5,
+                           interval=2)
+        # Now that we know that reservation is cleared we should
+        # check for time difference between each cancellation request
+        out = self.mom.log_match("%s;Canceling ALPS reservation *" % jid1,
+                                 n='ALL', max_attempts=5, regexp=True,
+                                 allmatch=True)
+
+        # We found something, Let's first check there are atleast 2 such
+        # log messages, If not then that means reservation was cancelled
+        # in the first attempt itself, at that point right thing to do is
+        # to either run it again or find out a way to delay the reservation
+        # cancellation at ALPS level itself.
+        if len(out) >= 2:
+            nline_prev, line_prev = out[0]
+            time_prev = self.get_epoch(line_prev)
+            for data in out[1:]:
+                nline, line_current = data
+                time_current = self.get_epoch(line_current)
+                self.assertGreaterEqual(time_current - time_prev,
+                                        math.ceil(ari),
+                                        msg="alps_release_interval not working"
+                                        )
+                time_prev = time_current
+
+        else:
+            self.skipTest("Reservation cancelled without retry, Try again!")
+
+    def test_alps_release_jitter(self):
+        """
+        Set alps_release_jitter to a higher value and then notice that
+        subsequest reservation cancellation requests are made by adding
+        a random time interval (less than jitter) to alps_release_interval.
+        """
+        arj = 2.198
+        ari = 1
+        max_delay = (ari + math.ceil(2.198))
```

Comment:
Either put arj in math.ceil() or just use 3.  I dislike using functions on literals when the return value is known up front.